hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/health/ECReplicationCheckHandler.java

```diff
@@ -71,7 +80,7 @@ public boolean handle(ContainerCheckRequest request) {
       }
       // TODO - if it is unrecoverable, should we return false to other
       //        handlers can be tried?
```

Comment:
Unrelated to this PR but I'm addressing the TODO. If it is unrecoverable I don't think we should return false now because the unhealthy handler will try to handle them (we did add a check in that handler to preserve them if it finds the container is under replicated so it _should_ be fine anyway).

Reply:
I have removed this TODO for now.